app/templates/participate/login.hbs

```diff
@@ -24,7 +24,7 @@
     <div class="row">
       <div class="col-md-12">
         <h1>International Situations Project</h1>
```

Comment:
Although not part of this PR- should the title have a translation string? Or are we intentionally keeping that hard-coded in english?